collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java

```diff
@@ -576,6 +575,8 @@ public void setTaskListener(TaskListener tl) {
             String requestString = "'" + currentDir + "' in parents and trashed=false";
             Files.List request = null;
 
+            //// TODO: 19/3/17 update library
+/*
```

Comment:
I'm wondering if we're gonna remove this? I'm not a big fan of commenting blocks of code 🤣